8Knot/pages/affiliation/visualizations/commit_domains.py

```diff
@@ -46,18 +64,30 @@
                 ),
                 dcc.Loading(
                     dcc.Graph(id=f"{PAGE}-{VIZ_ID}"),
+                    style={"marginBottom": "1rem"},
+                ),
+                html.Hr(  # Divider between graph and controls
+                    style={
+                        "borderColor": "#909090",
+                        "margin": "1.5rem -1.5rem",
+                        "width": "calc(100% + 3rem)",
+                    }
                 ),
                 dbc.Form(
                     [
                         dbc.Row(
                             [
-                                dbc.Label(
-                                    "Contributions Required:",
-                                    html_for=f"company-contributions-required-{PAGE}-{VIZ_ID}",
-                                    width={"size": "auto"},
-                                ),
                                 dbc.Col(
                                     [
+                                        dbc.Label(
+                                            "Contributions Required:",
+                                            html_for=f"company-contributions-required-{PAGE}-{VIZ_ID}",
+                                            style={
+                                                "fontSize": "12px",
+                                                "fontWeight": "bold",
+                                                "marginBottom": "0.5rem",
+                                            },
```

Comment:
are these one-off styles/exceptions for this element or can they be refactored to the stylesheet to apply to other things?